pkg/cmd/metacontroller/main.go

```diff
@@ -56,6 +56,7 @@ var (
 	leaderElectionNamespace    = flag.String("leader-election-namespace", "", "Determines the namespace in which the leader election resource will be created")
 	leaderElectionID           = flag.String("leader-election-id", "metacontroller", "Determines the name of the resource that leader election will use for holding the leader lock")
 	healthProbeBindAddress     = flag.String("health-probe-bind-address", ":8081", "The address the health probes endpoint binds to")
+	targetLabelSelector        = flag.String("target-label-selector", "", "Targeted labels for label selector used to limit metacontroller to watch for specific resources")
```

Comment:
@newtondev can we emphasis it is for Composite/Decorator controllers ?

Reply:
Thanks @grzesuav . I have updated accordingly based on your suggestion.